content/docs/2.7/providers/azure-workload.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Azure Workload Identity"
+layout = "provider"
```

Comment:
This is a new one so we can leave it out - Thanks!

Reply:
Done!